pkg/addons/helpers.go

```diff
@@ -42,3 +46,29 @@ func vSphereService() *corev1.Service {
 		},
 	}
 }
+
+// AreEmbeddedAddons checks if all specified addons are embedded addons
+func AreEmbeddedAddons(addons []kubeoneapi.Addon) (bool, error) {
```

Comment:
Would it be better if we call this function `EmbeddedAddonsOnly` instead?

Reply:
Sure thing let's call it that